bin/sentieon-genomics-202010.02/lib/python/sentieon/vcflib/tribble.py

```
@@ -0,0 +1,320 @@
+        self.tree = IntervalTree()
+        self.curr = [0, 0, off, off, 0] # [sloc, eloc, soff, eoff, count]
+
+    def decode(self, data, off):
+        chrom,_ = data[off:].split(b'\0',1)
+        off += len(chrom) + 1
+        self.chrom = chrom.decode()
+        s = struct.Struct('<i')
+        nitvs, = s.unpack_from(data, off)
+        off += s.size
+        s = struct.Struct('<iiQi')
+        for _ in xrange(nitvs):
```

Ruff (0.8.2)

`xrange` may be undefined, or defined from star imports

(F405)